cat-theme/src/css/header/_navigation.scss

```diff
@@ -0,0 +1,225 @@
+$background--color:#e6e6e6;
+$icon--color:#1e2023;
+$font--color:#ffffff;
+$font--color--active:#000000;
+$icon--size: 3rem;
+$font--primary:'Fira Sans',
+sans-serif;
+$transition--length: .8;
+
+#navigation {
+    align-items: center;
+    display: flex;
+    position: fixed;
+    justify-content: space-between;
+    width: 100%;
+    z-index: 2;
```

Comment:
For z-index values, we should follow the convention we have set at https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L1943...L1959.